UpdateDB/DataLoading/DataLoader.cs

```diff
@@ -18,6 +18,12 @@ public interface IDataLoader
         bool SavePathIsFolder { get; }
 
         /// <summary>
+        /// Sets the <see cref="HttpContent"/> used for downloading data.
```

Comment:
Did you mean HttpClient?
I see that it was previously called that in other places, but if it's intended I can't make sense of it.